src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -263,9 +263,9 @@ private function process_directives_args( string $html, array &$context_stack, a
 			 * We still process the rest of the HTML.
 			 */
 			if ( 'SVG' === $tag_name || 'MATH' === $tag_name ) {
-				if ( $p->get_attribute_names_with_prefix( 'data-wp-' ) || str_contains( $p->get_updated_html(), 'data-wp-' ) ) {
-					/* translators: 1: SVG or MATH HTML tag. */
-					$message = sprintf( '%1s tags are currently incompatible with the directives processing. Please refrain from adding directives to them.', $tag_name );
+				if ( $p->get_attribute_names_with_prefix( 'data-wp-' ) ) {
+					/* translators: %s: SVG or MATH HTML tag. */
+					$message = sprintf( '%s tags are currently incompatible with the directives processing. Please refrain from adding directives to them.', $tag_name );
```

Comment:
This is missing `__`, it should look more like this:

https://github.com/WordPress/wordpress-develop/blob/0c99bcfe863cf4c5239cecfc9b5a65acaf0134cd/src/wp-login.php#L71-L72